Easydict/Swift/Feature/Configuration/Configuration+Defaults.swift

```diff
@@ -201,7 +201,11 @@ extension Defaults.Keys {
         default: OpenAIUsageStats.default
     )
     static let openAIEndPoint = Key<String?>(EZOpenAIEndPointKey)
-    static let openAIModel = Key<OpenAIModel>(EZOpenAIModelKey, default: .gpt3_5_turbo_0125)
+    static let openAIModel = Key<String>(EZOpenAIModelKey, default: OpenAIModel.gpt3_5_turbo.rawValue)
+    static let openAIAvailableModels = Key<String?>(
+        "EZOpenAIAvailableModelsKey",
```

Comment:
It would be better to define a constant string like `EZCustomOpenAIModelssAvailableKey `

Reply:
Why? This value doesn't seem to be needed elsewhere.

Reply:
The main reason is to keep consistent with others definitions like `EZOpenAIModelKey ` and `EZOpenAIAPIKey`.  It will be easy to manage them in a single place like `EZConstKey`. 

And do we need to support this key in schema parse read & write functions?

Reply:
You're right, the current configuration of the OpenAI service is required to support command write modifications, so the `EZOpenAIAvailableModelsKey` needs to be open. I forgot about that.

But for some keys  that don't need to be external, such as `EZOpenAIValidModelsKey`, I think it's possible to just use literals?

Reply:
I believe using literals is acceptable, but I would still recommend defining a constant like others. As mentioned above, this approach would make it easier to use if it needs to be external or remove it if not used anymore in the future.

Reply:
ok